session1/examples/generics.ts

```diff
@@ -0,0 +1,33 @@
+//Generic type variables
+function checkPasswordLength<T>(password: T) {
+    console.log(password.length);
+}
+
+//Generic types
+function testArray<T>(item: Array<T>) {
+    return item;
```

Comment:
It doesn't really make sense to return the item just like that. If the function has been created, let's add some logic to it.